pom.xml

```
@@ -182,36 +188,37 @@
         <mongo.java.driver.version>3.6.4</mongo.java.driver.version>
         <mongo.scala.driver.version>2.7.0</mongo.scala.driver.version>
         <openui5.version>1.67.1</openui5.version>
-        <parquet.version>1.10.0</parquet.version>
+        <parquet.version>1.12.2</parquet.version>
         <requests.version>0.1.6</requests.version>
-        <scala.compat.version>2.11</scala.compat.version>
+        <scala.compat.version>2.12</scala.compat.version>
         <scala.java8.compat.version>0.9.0</scala.java8.compat.version>
-        <scala.version>2.11.12</scala.version>
-        <scalatest.maven.version>2.1.0</scalatest.maven.version>
-        <scalatest.version>3.2.14</scalatest.version>
+        <scala.version>2.12.12</scala.version>
```

```suggestion
        <scala.version>2.12.17</scala.version>
```
Why not use latest **maintenance** release?